tests/components/hue/test_light.py

```diff
@@ -524,8 +527,8 @@ def mock_bridge(hass):
     await hass.services.async_call(
         "light", "turn_on", {"entity_id": "light.group_1"}, blocking=True
     )
-    # 2x group update, 2x light update, 1 turn on request
-    assert len(mock_bridge.mock_requests) == 5
+    # 2x group update, 1x light update, 1 turn on request
+    assert len(mock_bridge.mock_requests) == 4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4927032)